### PR TITLE
Add the AutoFeat endpoint

### PR DESCRIPTION
**Pull request by Hiba-Alili**

### CODE REVIEW

**Review by dianajlailaty — APPROVED** [review]

**Review by mklkun — APPROVED** [review]

File: MachineLearning/resources/catalog/Import_Data_Interactive_Script.py [hunk 1]
```diff
@@ -111,8 +111,8 @@ def get_open_port():
 flask_port = get_open_port() # 8001
 bokeh_url = "http://"+external_ip+":"+str(bokeh_port)
 flask_url = "http://"+external_ip+":"+str(flask_port)
-print("bokeh_url: ", bokeh_url)
-print("flask_url: ", flask_url)
+
+schedulerapi.addExternalEndpointUrl(variables.get("PA_JOB_ID"), "AutoFeat", bokeh_url, "/automation-dashboard/styles/patterns/img/wf-icons/data-processing.png")
```
Comment (mboussaa) [inline MachineLearning/resources/catalog/Import_Data_Interactive_Script.py:115]: And flask ?

Comment (mboussaa) [reply MachineLearning/resources/catalog/Import_Data_Interactive_Script.py:115]: shouldn't we add schedulerapi.connect() before ?

Comment (Hiba-Alili) [reply MachineLearning/resources/catalog/Import_Data_Interactive_Script.py:115]: i already got the access to schedulerapi and acquired a session id before. Please check the line 48.

Comment (Hiba-Alili) [reply MachineLearning/resources/catalog/Import_Data_Interactive_Script.py]: Actually, we're embedding a bokeh content into a flask web app. The bokeh server should be running in the background to get its content and embed it into the flask app. So we need to provide access only to the flask app, instead.  Thank you for the remark.

**Review by mboussaa — APPROVED** [review]

**Pull request merged**